src/shared/config/sentry.ts

```
@@ -0,0 +1,33 @@
+import * as Sentry from '@sentry/react';
```

ESLint

[error] Replace `'@sentry/react'` with `"@sentry/react"`

(prettier/prettier)